CMakeLists.txt

```diff
@@ -940,7 +940,7 @@ IF(XNNPACK_TARGET_PROCESSOR STREQUAL "arm")
   # Related links:
   #   https://github.com/android/ndk/issues/910
   #   https://reviews.llvm.org/D58477
-  IF(ANDROID_NDK_MAJOR AND ANDROID_NDK_MAJOR LESS 21)
+  IF((ANDROID_NDK_MAJOR AND ANDROID_NDK_MAJOR LESS 21) OR OHOS)
```

Comment:
is OHOS defined on its own?  Shouldnt this be CMAKE_SYSTEM_NAME MATCHES OHOS like above?
This section is meant for old ndk builds for armv7 where the abi needs to be specified.  Is this true for all harmony builds/cpus?